renderers/main.py

```diff
@@ -153,7 +153,7 @@ def __draw_game(self):
                 self.data.scrolling_finished = True
 
         else:  # draw a live game
-            if scoreboard.homerun() or scoreboard.strikeout():
+            if scoreboard.homerun() or scoreboard.strikeout() or scoreboard.hit() or scoreboard.walk():
```

Comment:
Does hit and walk lead to flashing text like the strikeout does?

Reply:
Yes, they do. During the first draft I didn’t intend it, but I kind of liked it during testing and it grew on me, especially for home runs. I would be ok with it either way, but will defer to y’all.

Reply:
With home runs already having the bases circling animation it may be a bit much, but I’m happy to give it a try. 

It’s also totally reasonable for us to circle back and add more knobs for customization of this if we want to later